tools/abel/README.md

```diff
@@ -0,0 +1,127 @@
+# ABEL
+### Automated Building Entity Loader
+
+## Overview
+ABEL is a tool allowing system integrators to easily and efficiently create and modify [Building Configuration files](../../ontology/docs/building_config.md).
+
+## Setup and installation
+
+### Setting up a tooling environment
+
+Clone the Digital Buildings respository
+
+  ```
+  git clone https://github.com/google/digitalbuildings.git
+  ```
+
+Navigate to the Digital Buildings Repository's tooling library
+
+  ```
+  cd digitalbuildings/tools/
+  ```
+
+Create a virtual environment with an environment name, in this example: `tooling`
+
+  ```
+  virtualenv tooling
+  ```
+
+Activate your virtual environment
+
+* MacOs/Linux:
+
+  ```
+  source tooling/bin/activate
+  ```
+
+* Windows:
+
+  ```
+  tooling/Scripts/activate
+  ```
+
+Depending on your operating system run either of the global setup scripts to configure dependencies
+
+* Linux/MacOs:
+  ```
+  bash pip_install.sh
+  ```
+
+* Windows:
+
+  ```
+  pip_install.bat
+  ```
+
+### Obtain a spreadsheet token for your google sheets account
+
+1. Be added to a GCP service account with `Service Account User` and `Service Account Token Creator` roles.
+
+    * Please contact your IoT TPM for details and to be added to a project.
+
+2. Create a token request file, `token_request.json` with contents:
+
+    ```
+    {"scope": [    "https://www.googleapis.com/auth/spreadsheets" ],  "lifetime": "3600s"}
+    ```
+
+3. Run the below curl command to obtain a token:
+
+    ```
+        curl -X POST \
+        -H "Authorization: Bearer "$(gcloud auth application-default print-access-token) \
+        -H "Content-Type: application/json; charset=utf-8" \
+        -d @token_request.json \
+        "https://iamcredentials.googleapis.com/v1/projects/-/serviceAccounts/your-service-account@project-id.iam.gserviceaccount.com:generateAccessToken" >| spreadsheet_token.json
+    ```
+
+4. `spreadsheet_token.json` should look something like:
```

Comment:
I know this is a continued nit, but could you add verbiage here such as, "Confirm 'spreadsheet_token.json' looks similar to the below" ? (again, just for readability / consistency since it's an actionable step)

Reply:
cool, done.